cs/src/io/unsafe/MemoryHelper.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bond.IO.Unsafe
+{
+    internal unsafe class MemoryHelper
+    {
+        [DllImport("msvcrt.dll", EntryPoint = "memcpy", CallingConvention = CallingConvention.Cdecl, CharSet = CharSet.Unicode, SetLastError = false)]
```

Comment:
I would expect this to fail OS X test but it [https://travis-ci.org/Microsoft/bond/jobs/89251589](succeeded). Are we missing a test case? Are we missing a test case?

We need to keep the code cross-platform.